media.module

```diff
@@ -22,3 +23,38 @@ function media_help($route_name, RouteMatchInterface $route_match) {
     default:
   }
 }
+
+/**
+ * Implements hook_form_alter().
+ */
+function media_form_alter(&$form, FormStateInterface &$form_state) {
+  if (isset($form['#form_id'])) {
```

Comment:
Form ID should always be set. It is even 3rd argument to this function (see docs).